org.jacoco.build/pom.xml

```diff
@@ -394,6 +405,11 @@
           <artifactId>xml-maven-plugin</artifactId>
           <version>1.0</version>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>exec-maven-plugin</artifactId>
+          <version>1.6.0</version>
```

Comment:
@marchof why not simply use 1.5.0 version of exec-maven-plugin that requires Java 5?